SelfAndPeerEvaluations/partner_eval_quiz_final_grader.py

```diff
@@ -0,0 +1,101 @@
+import canvasapi
+from collections import defaultdict
+import datetime
+import json
+import os
+from utils import *
+
+
+class PartnerEvalQuizGrader:
+    def __init__(
+        self,
+        course: canvasapi.course.Course,
+        json_questions: dict,
```

Comment:
Can be resolved the [same way as here](https://github.com/mfbutner/CanvasHelpers/pull/10/files#r941853539)